xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -129,6 +129,28 @@ public BodyPart addPart(String mimeType, Object content, Map<String, Object> par
         return bodyPart;
     }
 
+    /**
+     * Add some content to the mail to be sent. Can be called several times to add different content type to the mail.
```

Comment:
i'd have added some explanation about why we need this method when it's not going to be possible to call it in Velocity (since you can't create a BodyPart in Velocity).